wallet-api-gen/README.md

```diff
@@ -0,0 +1,3 @@
+# wallet-api-gen
```

Comment:
TBH I think it's fine to have these as an exposed module of `wallet-api`, no need for another package.

Reply:
I separated it out because of the hedgehog dependency but I'm fine with leaving the module in `wallet-api`

Reply:
I say we don't bother until someone complains.